tests/cortex_cli_test.py

```diff
@@ -598,7 +598,7 @@ def test_auth_logout_fails_by_server_response_no_pid(credentials, config_dict, t
             '--config-file', 'config.json',
             '--username', credentials['username'],
             '--password', credentials['password'],
-            '--no-daemon', # do not daemonize
+            '--no-refresh', # do not start token manager
```

Comment:
Do we now have any tests left that are not using the `--no-refresh` option?

Reply:
No, we don't.

Not using the `--no-refresh` option requires running a separate process. Perhaps, it's a responsibility of integration tests or even e2e tests?

Reply:
Well, there is a test for the daemon mode. Would be nice to have a test for `--no-daemon` and no `--no-refresh` mode as well.

Reply:
For posterity: talked afk, decided to consider proper (real) daemon testing in integration tests.